docs/getting_started.rst

```diff
@@ -29,6 +29,7 @@ Add `beam`, `beam.themes.boostrap4` and `crispy_forms` to your `INSTALLED_APPS`
 Quickstart example
 ------------------
 Let's say you have a list of books that you want to manage in your Django project. You can use django-beam to quickly create a list view, detail view, create view and update view for your books.
+Note that your test app must not have an ``app_name`` in `urls.py`.
```

Comment:
I think you need to set `url_namespace` on the viewset accordingly if you set an `app_name`, is that right?